### PR TITLE
PP2 CH17

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: piety-promoted-v2/modernized/18-chapter-17.adoc [hunk 1]
```diff
@@ -406,7 +409,13 @@ for it is he that has plucked my feet out of the miry clay,
 and set them upon a rock that was higher than I; wherefore to him shall the honor, glory,
 and praise be given, who is over all worthy forever and evermore.
 
-Signed--Thomas Mawditt, 1748.`'
+[.signed-section-closing]
+Signed
+
+[.signed-section-signature]
+Thomas Mawditt 1748
```
Comment (maijapapaya) [inline piety-promoted-v2/modernized/18-chapter-17.adoc]: Not sure if the date should just go by his name since there isn't any place he listed too?

Comment (jaredh159) [reply piety-promoted-v2/modernized/18-chapter-17.adoc]: let's just remove the "Signed" part entirely, and then move the date down below on it's own as a `[.signed-section-context-close]`

File: piety-promoted-v2/modernized/18-chapter-17.adoc [hunk 2]
```diff
@@ -416,4 +425,5 @@ under a well-grounded apprehension that his day`'s work was done.
 He departed this life the 13th of the Seventh month, 1770; aged eighty-one,
 and had been a minister about forty-eight years.
 
+[.the-end]
```
Comment (jaredh159) [inline piety-promoted-v2/modernized/18-chapter-17.adoc]: excellent use of `[.the-end]` 👍

**Comment by jaredh159** [comment]
looks good, thanks! merging.

**Pull request merged**